app/src/main/java/org/simple/clinic/home/patients/PatientsTabUi.kt

```diff
@@ -7,4 +7,5 @@ interface PatientsTabUi {
   fun showIllustration()
   fun showUserStatusAsPendingVerification()
   fun hideUserAccountStatus()
+  fun showUserStatusAsWaiting()
```

Comment:
Maybe we should maintain consistency with `showUserStatusAsPendingVerification` and rename this to `showUserStatusAsWaitingForApproval`